packages/edit-post/src/components/keyboard-shortcuts/index.js

```diff
@@ -168,6 +173,18 @@ function KeyboardShortcuts() {
 		{ bindGlobal: true }
 	);
 
+	useShortcut(
```

Comment:
Why do we need this now, didn't we have something similar before?

Reply:
Yes, it's embedded into the [dropdown toggle of the `BlockNavigation` component](https://github.com/WordPress/gutenberg/blob/0d1757f3f43a76000b804e561acc7ed6a9c57e8f/packages/block-editor/src/components/block-navigation/dropdown.js#L27-L34), which is not used here anymore.

`BlockNavigationDropdown` contained in itself both the toggle button and the actual list view rendered in a dropdown, so it made sense to have the shortcut inside it as well.
The persistent list view though lives in a different place than its toggle button, and so the shortcut needs to be moved to the global ones.

FYI the dropdown toggle component is still used elsewhere (AFAICS only in the `edit-widgets`).

Reply:
It seems weird, that the dropdown which is in block-editor relies on a shortcut defined in edit-post. Let's remove the useShortcut call in the dropdown for now. We can't remove the component yet, maybe we can deprecate it

Reply:
I'd do it in a separate PR though, as it might affect consumers using the dropdown (namely, `edit-widgets`).

Reply:
it won't since that shortcut is not registered in the first place in edit-widgets, so it just does nothing.